env.sample

```diff
@@ -7,6 +7,8 @@ export PH_VOU_CALLBACK_URL=https://webhook.site/
 
 PH_ACT_BACKEND_SERVER_URL=https://paymenthub.qa.oneacrefund.org/opsapp/api/v1
 
+export PH_G2P_PAYMENT_CONFIG_URL=https://gateway.mifos.community/v1.0/g2p-payment
```

Comment:
This variables to which backend service must point?

Reply:
This is a new service containing g2p configuration with operations like fetch govt entity, fsps , programs and g2p config.
Basically previously the g2p config was coming from a yaml file under budget account 

https://github.com/openMF/ph-ee-bulk-processor/blob/9bf984fcf16105081ff53936f72405814ef8ba71/src/main/resources/application.yaml#L160-L161C1
So I have developed a new API as suggested by my mentor for it so we can configure it properly instead of changing the yaml file.

Currently it's working independently as a new service 
In future we need to connect it with payment bulk processing